File: Python/SoftUni PB/Exercises/Python Basics/First Steps in Coding - Exercise/Food Delivery.py
chicken_menu = 10.35
fish_menu = 12.4
vegetarian_menu = 8.15
delivery_bill = 2.5
chicken_menu_number = int(input())
fish_menu_number = int(input())
vegetarian_menu_number = int(input())
price = (chicken_menu_number * 10.35) + (fish_menu_number * 12.4) + (vegetarian_menu_number * 8.15)
dessert = (20/100 * price)
total = price + dessert + 2.5
print(total)

File: JS-Basics/First Steps/Exercise/foodDelivery.js
function foodDelivery(input) {
    let chickenMenu = 10.35;
    let fishMenu = 12.4;
    let vegetarianMenu = 8.15;

    let chickenMenusAmount = Number(input[0]);
    let fishMenusAmount = Number(input[1]);
    let vegetarianMenusAmount = Number(input[2]);

    let chickenPrice = chickenMenu * chickenMenusAmount;
    let fishPrice = fishMenu * fishMenusAmount;
    let vegetarianPrice = vegetarianMenu * vegetarianMenusAmount;

    let price = chickenPrice + fishPrice + vegetarianPrice;
    let dessertPrice = 0.2 * price;

    let deliveryTax = 2.5;
    
    let totalPrice = price + dessertPrice + deliveryTax;

    console.log(totalPrice);

}

foodDelivery(['2', '4', '3']);
